src/cr/cube/matrix/measure.py

```diff
@@ -1253,6 +1257,14 @@ def _df(self):
         Na, Nb, Nab = self._valid_counts
         return Na + Nb - Nab
 
+    @lazyproperty
+    def _proportions(self):
+        """tuple(float64) of column proportions"""
```

Comment:
I would expand the docstring of this property, TBH I'd like to see what `pa` and `pb` are  also a description on these `pa/Sa` and `pb/Sb` divisions